src/background/services/globalConfigService.ts

```diff
@@ -312,6 +316,7 @@ export default class GlobalConfigService {
       },
     ]
     config.myDapplets = []
+    config.connectedAccountsContractAddress = 'dev-1659683929908-36510272337320'
```

Comment:
We need to deploy the contract to non-dev address like `v1.example.testnet`

Reply:
Should we deploy it on the mainnet?

Reply:
exactly!